rotary/rules/annotation.smk

```diff
@@ -172,7 +175,8 @@ rule run_dfast:
         "{sample}/benchmarks/annotation/annotation_dfast.txt"
     params:
         db=directory(os.path.join(DB_DIR_PATH,"dfast_" + VERSION_DFAST)),
-        strain='{sample}'
+        strain='{sample}',
+        no_functional_annotation="--no_func_anno" if not ANNOTATION_MAP.dfast_func else ''
```

Comment:
When I tested DFAST in `--no_func_anno` mode, I found that the DB directory (i.e., `--dbroot`) does not need to be specified in this mode. This is nice, because it means that a user could install rotary and use it in `--no_func_anno` mode without needing to download any large annotation DBs for DFAST. If possible, it would be great if we could add the ability to skip the `--dbroot` argument in the shell command when DFAST runs in `--no_func_anno`.

Required changes:
- Add an if-else statement to the `db` param similar to what is used for the `no_functional_annotation` param. If `dfast_func` isn't specified, then `''` is returned, but otherwise `--dbroot {path to the db}` could be returned. The shell command would need to be modified to replace `--dbroot {params.db}` with just `{params.db}`.
- Only require the `install_finished` input if `dfast_func` is specified.
  - For clarity, `install_finished` could be renamed `annotation_db_install_finished` or something similar
  
@LeeBergstrand This seems relatively straightforward to implement -- thoughts?

Reply:
Let me know if it would be easier for me to push a commit for this.

Reply:
@jmtsuji Addressed

Reply:
Good catch!